### DISCUSSION TITLE
Lazy loading of components

### DISCUSSION THREAD

**Discussion by iabhiyaan** [GrapesJS/grapesjs#3871] [Q&A]
Hello @artf, 
Is there is any way for lazy loading of components while loading from storage manager ?

It takes more than 14 sec to render template in the canvas from storage manager.

```js
storageManager:{
urlLoad: https://endpoint.com/load-template/1
}
```

**Comment by artf** [comment]
GrapesJS frames work exactly like any other browser page, so if you have a huge blocking resource to load, that will block the rendering.
